packages/gestalt/src/ButtonToggle.css

```diff
@@ -0,0 +1,91 @@
+.button {
+  composes: borderBox minWidth60 from "./Layout.css";
+  composes: border from "./Borders.css";
+  border-radius: var(--rounding-600);
+}
+
+.parentButton {
+  composes: paddingX0 paddingY0 from "./boxWhitespace.css";
+  composes: noBorder from "./Borders.css";
+  background-color: var(--color-transparent);
+}
+
+.noBorder {
+  composes: noBorder from "./Borders.css";
+}
+
+.childrenDiv {
+  composes: flex from "./Layout.css";
+  composes: xsItemsCenter justifyCenter from "./Layout.css";
+  composes: fullWidth from "./TapArea.css";
+}
+
+.sm {
+  composes: small from "./Layout.css";
+  composes: paddingX3 paddingY1 from "./boxWhitespace.css";
+}
+
+.md {
+  composes: medium from "./Layout.css";
+  composes: paddingX3 paddingY2 from "./boxWhitespace.css";
+}
+
+.lg {
+  composes: large from "./Layout.css";
+  composes: paddingX4 paddingY3 from "./boxWhitespace.css";
+}
+
+.enabled {
+  composes: pointer from "./Cursor.css";
+}
+
+/* STATES */
+
+.disabled {
```

Comment:
disabled can be consolidated

.disabled { 
  composes: borderColorLightGrayDisabled from "./Borders.css";
  cursor: default;
  pointer-events: none;
}

.disabledTransparent { 
background-color: var(--color-background-button-tertiary-default);)
}

.disabledRed { 
 background-color: var(--color-background-button-disabled-default);
}

Reply:
DONE